evm_loader/solana_utils.py

```diff
@@ -205,7 +204,7 @@ def ether2program(self, ether):
         if isinstance(ether, str):
```

Comment:
You should get the implementation of `ether2programAddress` for `ether2program` function.
Also drop `ether2programAddress` function.